src/Symfony/Component/Translation/Catalogue/OperationInterface.php

```diff
@@ -48,7 +48,7 @@ public function getDomains();
      *
      * @return array
      */
-    public function getMessages($domain);
+    public function getMessages(string $domain);
```

Comment:
You can remove `@param string $domain` from the docblock now because that annotation has become redundant now.

Reply:
Removed docblock comments that have no additional value to the readability or the usability of the class.